features/steps/ruby_notifier_steps.rb

```diff
@@ -12,4 +12,30 @@
   steps %Q{
     When I set environment variable "#{env_var}" to "#{credentials}@#{current_ip}:#{MOCK_API_PORT}"
   }
+end
+Then(/^the "(.+)" of the top project stackframe equals "(.+)"(?: for request (\d+))?$/) do |element, value, request_index|
```

Comment:
Can we have a blank line between step definitions for organization/easier editor navigation? (between lines 15-16, 28-29)